src/Acuminator/Acuminator.Tests/Tests/Refactorings/ChangeEventHandlerSignatureToGeneric/ChangeEventHandlerSignatureToGenericTests.cs

```diff
@@ -13,64 +13,64 @@
 
 namespace Acuminator.Tests.Tests.Refactorings.ChangeEventHandlerSignatureToGeneric
```

Comment:
@SENya1990 do we want to merge this file with commented tests?

Reply:
No, I missed it, good that you noticed

Reply:
@metallmonster fixed, one more try